nav2_bt_navigator/src/navigators/navigate_to_pose.cpp

```diff
@@ -209,18 +209,29 @@ NavigateToPoseNavigator::initializeGoalPose(ActionT::Goal::ConstSharedPtr goal)
     feedback_utils_.global_frame, feedback_utils_.robot_frame,
     feedback_utils_.transform_tolerance);
 
+  geometry_msgs::msg::PoseStamped goal_pose;
+  if (!nav2_util::transformPoseInTargetFrame(
```

Comment:
A change here should also be made in the Navigate Though Poses navigator as well